bochiedev/SIL_Store
# Ch setup CICD

#### What does this PR do?
adds configurations for the CI CD pipeline (codecov, circleci, pre commit hooks)

#### Description of Task to be completed?
ensure a CI CD pipeline is set up 

#### How should this be manually tested?
This can be tested by pushing changes to Github and creating a pull request to the main branch to check if the tools work

#### Any background context you want to provide?

#### What are the relevant JIRA stories?

#### Screenshots (if appropriate)

#### Questions:
